packages/squiggle-lang/src/rescript/Utility/Operation.res

```diff
@@ -86,6 +93,8 @@ let logarithm = (a: float, b: float): result<float, Error.t> =>
     Ok(0.)
   } else if a > 0.0 && b > 0.0 {
     Ok(log(a) /. log(b))
+  } else if a == 0.0 {
+    Error(NegativeInfinityError)
```

Comment:
Have you considered returning negative infinity instead of an error? There are cases where this value is useful.